app/org/sagebionetworks/bridge/play/controllers/SurveyController.java

```diff
@@ -239,13 +239,13 @@ public Result updateSurvey(String surveyGuid, String createdOnString) throws Exc
         return okResult(new GuidCreatedOnVersionHolderImpl(survey));
     }
     
-    public Result publishSurvey(String surveyGuid, String createdOnString) throws Exception {
+    public Result publishSurvey(String surveyGuid, String createdOnString, String newSchemaRev) throws Exception {
         UserSession session = getAuthenticatedSession(DEVELOPER);
         StudyIdentifier studyId = session.getStudyIdentifier();
          
         Survey survey = getSurveyWithoutCacheInternal(surveyGuid, createdOnString, session);
-        
-        survey = surveyService.publishSurvey(studyId, survey);
+
+        survey = surveyService.publishSurvey(studyId, survey, Boolean.parseBoolean(newSchemaRev));
```

Comment:
In terms of backwards compatibility, does this return false if newSchemaRev is null?